pkg/app/gen/html.go

```diff
@@ -1284,6 +1284,11 @@ var attrs = map[string]attr{
 	},
 
 	// K:
+	"key-value": {
```

Comment:
`key-value` is too generic and does not necessarily represent an attribute. Call this `attribute` and move it under below the `A`.